src/main/java/mate/academy/store/model/User.java

```diff
@@ -38,4 +47,47 @@ public class User {
 
     @Column(nullable = false)
     private boolean isDeleted = false;
+
+    @ManyToMany(fetch = FetchType.EAGER)
```

Comment:
```suggestion
    @ManyToMany
```

use `@EntityGraph(attributePaths = "roles")` in repo instead

Reply:
Not fixed